sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1717,13 +1717,13 @@ public virtual Response<RecognizeLinkedEntitiesResultCollection> RecognizeLinked
         internal virtual TextDocumentInput ConvertToDocumentInput(string input, string language, int id = 0)
             => new TextDocumentInput($"{id}", input) { Language = language ?? _options.DefaultLanguage };
 
-        private List<TextDocumentInput> ConvertToDocumentInputs(IEnumerable<string> inputs, string language)
+        internal virtual List<TextDocumentInput> ConvertToDocumentInputs(IEnumerable<string> inputs, string language)
             => inputs.Select((input, i) => ConvertToDocumentInput(input, language, i)).ToList();
 
-        private DetectLanguageInput ConvertToDetectLanguageInput(string input, string countryHint, int id = 0)
+        internal virtual DetectLanguageInput ConvertToDetectLanguageInput(string input, string countryHint, int id = 0)
```

Comment:
In my opinion we don't need to test every method on the client, if there are public methods that call into these having tests for public ones is enough.

Reply:
noted. Thank you!